docs/HOWTO.md

```diff
@@ -1,45 +1,34 @@
-How to create and track horizontal issues
-=========================================
+For Working Groups
+==================
 
-# What is a horizontal issue?
+* Apply the *-tracker label in your own repository to draw a horiztontal review group's attention to an issue in one of your own repositories.  Horizontal review groups may also apply the label if they are interested in tracking a particular issue.
 
-Horizontal issues are issues on specifications raised by an [horizontal group](https://www.w3.org/wiki/DocumentReview#How_to_get_horizontal_review).
+* Horizontal review groups may apply the *-needs-resolution to issues they expect to be resolved before the specification moves to a new maturity level.  
 
-Those issues are tracked more closely due to their potential wide impact on users, organizations or the Web architecture.
+* If *-needs-resolution is applied to an issue, automatic tooling will remove the *-tracker label.
 
-# Horizontal issues and W3C Process
+## Do these labels replace the need to ask for horizontal review?
 
-Horizontal issues should be raised and tracked as early possible. Some get raised before a specification is adopted by a W3C Working Group to be on the Standard track.
+No.  The is *-tracker label asks for input on a specific issue.  For through review, follow [the published process for requesting horitzontal review](https://www.w3.org/wiki/DocumentReview#How_to_get_horizontal_review)
 
-From a W3C Process perspective, the Director pays additional attention to horizontal issues before approving a new maturity level for a specification.
 
-Note: In the implementation of the Process 2020, the importance of an issue will more directly impact whether a Group is allowed to get an automatic approval from the Director.
+For horizontal review groups
+============================
 
-# How do W3C track horizontal issues?
+* WGs may apply the *-tracker label to issues in their own repositories to draw your attention to specific issues. 
 
-W3C tracks horizontal issues using a set of GitHub labels, allowing both specification contributors and horizontal reviewers to flag an issue as an horizontal issue. The level of importance of such issue may vary depending on the opinion of an horizontal group.
+* You may also apply the *-tracker label to issues you want to follow.
 
-# What's the difference between -tracker and -needs-resolution?
+* Apply the *-needs-resolution label to issues you expect to be resolved before a specification moves to a new maturity level.
```

Comment:
A *-tracker label can be replaced with a *-needs-resolution label if the horizontal group wants to see a specified resolution before the spec moves to the next maturity level (or vice-versa, if the resolution is no longer deemed important).  However, there should only one of *-tracker or *-needs-resolution labels, not both, and the labels should be changed both on the Working Group repo and the HR group's tracker repo.

Reply:
This may belong further down, were you have 
>  * Alternatively, you may apply the needs-resolution label to an existing tracking issue to upgrade it.

Reply:
My understanding was that the tooling allows us to update either repo and the change will propagate.  Is that not the case?

Reply:
It could be, but i'm not aware of it. @plh ?  As mentioned elsewhere, however, i hope it's NOT the case for a change from needs-res to tracker, since that allows WGs to game the system without us noticing (unintentionally, i'm sure).

Reply:
I've been updating the tooling to eliminate tracker if needs-resolution is present.

Reply:
How does it work in the following scenarios?
1. The WG repo has needs-resolution, but not the HR repo.
2. Vice versa.
3. One or more of the above have both a tracker and needs-res label.

I guess this means that if someone demotes an issue to tracker they must make the change in both repos, otherwise the needs-res will win out(?)